packages/sdk/schema/src/formatting.ts

```diff
@@ -4,6 +4,83 @@
 
 import { FieldValueType } from './types';
 
+/**
+ * Parse value by field value type.
+ * Used by Table, Sheet, etc.
+ * Handles various data types including booleans, numbers, dates, and strings.
+ * Returns undefined for empty or null inputs.
+ */
+export const parseValue = (type: FieldValueType, value: any) => {
```

Comment:
this might be better in react-ui-data since it's UX?

Reply:
Yeah I was thinking that for the `cellClassesForFieldType` function as well.